pom.xml

```diff
@@ -46,7 +47,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <optional>true</optional>
+            <version>${org.projectlombok.version}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
```

Comment:
Still security leftovers. Keep track of what you are adding to this pom

Reply:
I will try.

Reply:
Commented back but didint remove it

Reply:
THATS STILL HERE